src/antsibull_changelog/plugins.py

```diff
@@ -79,12 +79,19 @@ def from_dict(
 
         for plugin_type, plugin_data in data.items():
             for plugin_name, plugin_details in plugin_data.items():
+                if plugin_details["description"]:
+                    if plugin_details["description"].endswith("."):
+                        description = plugin_details["description"]
+                    else:
+                        description = plugin_details["description"] + "."
+                else:
+                    description = plugin_details["description"]
```

Comment:
This can be rewritten as follows:
```suggestion
                description = plugin_details["description"]
                if description and not description.endswith("."):
                    description += "."
```